python/marvin/contrib/vacs/mangagema.py

```diff
@@ -0,0 +1,66 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-10-11 17:51:43
+# @Last modified by:   Brian Cherinka
+# @Last Modified time: 2018-11-29 17:23:15
+
+from __future__ import print_function, division, absolute_import
+
+import marvin.tools
+
+from .base import VACMixIn
+
+
+class GEMAVAC(VACMixIn):
+    """Provides access to the MaNGA-GEMA VAC.
+
+    VAC name: GEMA
+
+    URL: link
+
+    Description: Returns GEMA VAC
```

Comment:
Can you write a brief description for your VAC here, and change the URL link to point to the DR15 documentation page for your VAC?